content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Using OpenTelemetry Environment Variables with Datadog SDKs
+further_reading:
+    - link: '/tracing/trace_collection/library_config/dotnet-core'
+      tag: 'Documentation'
+      text: '.NET Core SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/dotnet-framework'
+      tag: 'Documentation'
+      text: '.NET Framework SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/go'
+      tag: 'Documentation'
+      text: 'Go SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/java'
+      tag: 'Documentation'
+      text: 'Java SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/nodejs'
+      tag: 'Documentation'
+      text: 'Node.js SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/php'
+      tag: 'Documentation'
+      text: 'PHP SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/python'
+      tag: 'Documentation'
+      text: 'Python SDK Configuration'
+    - link: '/tracing/trace_collection/library_config/ruby'
+      tag: 'Documentation'
+      text: 'Ruby SDK Configuration'
+---
+
+Since the Datadog SDKs implement the OpenTelemetry Tracing APIs and are continually being updated to further implement additional OpenTelemetry APIs, you can replace the OpenTelemetry SDK in your application with the Datadog SDK to receive traces and additional Datadog-supported signals with minimal changes to SDK configuration when using the OpenTelemetry SDK configurations outline below.
+
+SDK settings can be configured with the following environment variables with their equivalent Datadog environment variable mapping:
+
+Note: If both set, Datadog environment variables take precedence over OpenTelemetry environment variables, Likewise Datadog defaults take precedence over OpenTelemetry defaults. Please refer to the configuration page for the relevant SDK for more information on default values.
+
+[for more info, see the OpenTelemetry documentation][9]
+## General SDK Configuration
+
+`OTEL_SERVICE_NAME`
+: **Corresponding Datadog Environment Variable**: `DD_SERVICE`<br>
+Sets the service name<br>
+**Notes**: If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then `OTEL_SERVICE_NAME` takes precedence<br>
+
+`OTEL_LOG_LEVEL`
+: **Corresponding Datadog Environment Variable**: `DD_LOG_LEVEL`<br>
+Log level used by the SDK logger<br>
+**Notes**: A log level of debug will also map to `DD_TRACE_DEBUG=true`<br>
+This maps to `DD_TRACE_LOG_LEVEL` in `nodejs` & `php` sdk's<br>
+**Not Supported In**: `python`, `dotnet`, `ruby`, & `golang` sdk's<br>
```

Comment:
```suggestion
**Not Supported In**: Python, .NET, Ruby, and Go SDKs<br>
```
I think the monospace formatting for SDK names makes it a little harder to read. Also, we should capitalize SDK and not use an apostrophe.